core/src/main/scala/com/microsoft/azure/synapse/ml/fabric/OpenAITokenLibrary.scala

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.fabric
+
+import com.microsoft.azure.synapse.ml.logging.SynapseMLLogging
+import spray.json.DefaultJsonProtocol.StringJsonFormat
+
+object OpenAITokenLibrary extends SynapseMLLogging with AuthHeaderProvider {
+  var MLToken: Option[String] = None;
+  var IsMWCTokenEnabled: Boolean = true;
+  val BackgroundRefreshExpiryCushionInMillis: Long = 5 * 60 * 1000L
+  val OpenAIFeatureName = "SparkCodeFirst"
```

Comment:
make these private to encapsulate mutable state and avoid people being able to mess it up